packages/blob/src/put-helpers.ts

```diff
@@ -85,6 +91,12 @@ export async function createPutOptions<
     throw new BlobError('pathname is required');
   }
 
+  if (pathname.length > MAXIMUM_PATHNAME_LENGTH) {
```

Comment:
is this function also used by copy? If not we need to add the check to the copy method aswell

Reply:
indeed, I'll add that now

Reply:
Added in cdc40b753adb9f4610e37beb3d52291c517f0eab